functions.go

```diff
@@ -0,0 +1,38 @@
+package goPackage
+
+func Fun1(s string, slice []string) bool {
+	for _, sliceItem := range slice {
+		if sliceItem == s {
```

Comment:
In Go we use elimination way to write code means eliminating condition from top to bottom, This will reduce code block
`
for _, sliceItem := range slice{
 if sliceItem != s {
    continue
 } 

return true
 
 }